README.md

```diff
@@ -40,3 +40,54 @@ vagrant up
 This will start to create the boxes with ansible as a provisioned of the configuration.
 
 and that's it!, you can check if the nginx is running going to your browser and put 10.0.0.10 or 10.0.0.11
+
+# Terraform-AWS + ansible deploy.
+
+First of all you need to install terraform on your local machine.
+
+Terraform https://www.terraform.io/downloads.html
+
+After that go to AWS console and create your IAM user to get your aws_secret_key and your aws_access_key, and create your key pair too we will use it later for ssh connection.
+
+Now we are going to clone or download the repository of github.
+
+$ git clone https://github.com/graidelak/Techtest.git
+
+Now you have to copy the key pair that you created before in AWS inside the github repository folder.
+
+Now you have to edit this line in the file variables.tf
+
+variable "aws_access_key" {
+  default = ""
+  description = "the user aws access key"
+}
+variable "aws_secret_key" {
+  default = ""
+  description = "the user aws secret key"
+}
+
+and put in "default" the information of your aws-access-key and you aws-secret-key , this will permit terraform login on your console in AWS.
```

Comment:
It's better to use environment variables instead of editing the file. It avoids committing the file by mistake.